.github/workflows/docker-publish.yml

```diff
@@ -33,7 +33,24 @@ jobs:
 
     strategy:
       matrix:
-        image_version: ["image-gammapy", "image-gammapy-env-dask", "image-datascience-notebook", "image-gammapy-noir"]
+        include:
+          - image_version: datascience-notebook
+            singularity: false
+
+          - image_version: gammapy
+            singularity: false
+            gammapy_tag: 1.2
```

Comment:
so you'd need to make sure this value matches that in the requirements?

Reply:
Yes, this would be the idea.

Reply:
I had in mind that maybe we specify here only just the software name.
And pick the version from the docker image requirements or ARG.
Less redundancy, less opportunity for mistake.
Do you think this is feasible?

Reply:
Hum, I think that it's feasible, there is even 2 ways.

1. We provided the arg to the docker image build command
2. We parse the docker file to extract a predefined ARG variable

Could we do that in another PR ?

Reply:
> Hum, I think that it's feasible, there is even 2 ways.
> 
>     1. We provided the arg to the docker image build command
> 
>     2. We parse the docker file to extract a predefined ARG variable

I'd put it in the Dockerfile like `LABEL ch.ctaodc.gammapy_version=$gammapy_version` - will also add useful metadata. It will set both the default and provided arg.
And then extract it from the image when making a tag.

> Could we do that in another PR ?

Yes, as long as we make an issue and reference to it here.

But why, you need these images right away?

Reply:
Alright, so as discussed I made the change to provide the version for gammapy through env variables.